src/app/components/SubmitBar/DeltaStarCount.tsx

```diff
@@ -0,0 +1,46 @@
+import { faStar } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import * as styles from 'app/styles/DeltaStarCount.module.css';
+import * as DeltaStarCountInterfaces from 'app/types/DeltaStarCount';
+
+import classnames from 'classnames';
+import * as React from 'react';
+
+export class DeltaStarCount extends React.Component<DeltaStarCountInterfaces.Props, {}> {
+  constructor(props: DeltaStarCountInterfaces.Props) {
+    super(props);
+  }
+
+  public render() {
+    return <span className={classnames(styles.starSpan)}>{this.mapStars()}</span>;
+  }
+
+  private whiteStars() {
+    const { rating } = this.props;
+    const maxRating = 3;
+    return maxRating - rating;
+  }
```

Comment:
What use is this function? These three lines don't warrant a need for a function